context_lab/src/components/Weather.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from "react";
+import "../css/Weather.css";
+import apiKey from "../secret";
+import axios from "axios";
+import WeatherDisplay from "./WeatherDisplay";
+
+export const farenheit = temp => {
+  return (temp - 273.15) * 1.8 + 32;
+};
+
+const Weather = () => {
+  const [location, setLocation] = useState([]);
```

Comment:
A lot of these states can be set in one state object where you can just create one key for each of these states